src/main/java/lotto/LottoApplication.java

```diff
@@ -1,26 +1,38 @@
 package lotto;
 
 import lotto.data.Lotto;
+import lotto.data.LottoWinningPrice;
 
+import java.util.ArrayList;
 import java.util.List;
+import java.util.Map;
 
-import static lotto.view.InputView.getPurchaseAmount;
-import static lotto.view.InputView.getWinningNumbers;
+import static lotto.data.Lotto.getNewLottoList;
 import static lotto.service.LottoGame.*;
+import static lotto.view.InputView.*;
 import static lotto.view.ResultView.*;
 
 public class LottoApplication {
 
     public static void main(String[] args) {
         int purchaseAmount = getPurchaseAmount();
-        int amount = getAmountOfLotto(purchaseAmount);
-        viewAmount(amount);
+        int autoLottoCount = getAmountOfLotto(purchaseAmount);
+        int manualLottoCount = getManualLottoCounts();
+        List<Lotto> manualLottoList = getManualLottoNumbers(manualLottoCount);
+        viewAmount(autoLottoCount, manualLottoCount);
 
-        List<Lotto> lottoList = getLottoList(amount);
-        viewLottoList(lottoList);
+        List<Lotto> autoLottoList = getNewLottoList(autoLottoCount);
```

Comment:
<img width="1442" alt="스크린샷 2023-05-21 오후 8 15 36" src="https://github.com/next-step/java-lotto/assets/12059592/32923fa6-17bb-49df-a3b2-add511b93066">

현재 구조가 객체지향 패턴을 적용하고 있는것 일지 고민해보아요.
`Lottos`라는 일급콜렉션 객체를 반환해보는건 어떨까요?

이러한 구조로 표현될수 있을것 같아요.

```
public class Lottos {

    private static final int LOTTO_PRICE = 1000;

    private final List<Lotto> values;

    public static Lottos of(autoCount, manualCount) {
          # 생성
    }
```
> 이러한 관점으로도 볼 수 있다 정도로 봐주시고 다양하게 고민해주세요!

`Lottos` 구입한 로또를 가진 개념으로 접근한다면, `수익률`,`랭킹 데이터` 등의 책임을 부여 해 볼 수 있을것 같구요! 🤔 
구입에 대한 책임도 `Lottos` 가 직접 하도록 책임을 줄 수도 있구요!

```suggestion
        Lottos lottos = Lottos.of(수동_갯수,자동_갯수);
```

---

객체지향이 항상 답은 아니라고 생각합니다.  
하지만, 이번 과정에서는 객체지향,TDD 를 배워보는 과정인 만큼 객체지향 패턴을 적용해보면 좋을듯해요!